index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/home/safeez/Desktop/Portfolio/Portfolio-Website/style.css">
+    <title>Portfolio-Website</title>
+</head>
+<body>
+    <div class="header container">
+        <div class="header one">My Logo</div>
+
+        <div class="header menu-icon">
+            <img src="src/Icon-Menu.png" alt="menu"/>
+        </div>
+    </div>
+
+    <section class="heading">
+        <h1 class="title">I’m Shaheda
+            Glad to see you!</h1>
+    </section>
+
+    <section class="paragraph">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </section>
+
+    <p class="line">LET’S CONNECT</p>
+
+    <section class="social-media-icons">
```

Comment:
- [ ] Please ensure to combine these divisions in one section tag and the content is aligned using flexbox.